homework-3/.idea/encodings.xml

```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project version="4">
+  <component name="Encoding">
+    <file url="file://$PROJECT_DIR$/src/main/java" charset="UTF-8" />
+    <file url="file://$PROJECT_DIR$/src/main/resources" charset="UTF-8" />
+  </component>
+</project>
```

Comment:
No need to add such files, like compiler.xml, encodings.xml and etc. to the commit, because there are no business logic and it leads to bad readability while reviewing the code.